src/Components/Person/Person.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    {name && (
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+    )}
+
+    {age && (
+      <p className="Person__age">{`I am ${age}`}</p>
+    )}
+
+    {isMarried ? (
+      <p className="Person__partner">
+        My
+        {` ${sex === 'm' ? 'wife' : 'husband'}'s `}
+        name is
+        {` ${partnerName}`}
+      </p>
+    ) : (
```

Comment:
It's hard to understand this code with many conditional statements
Let's create variable for storing the result of this check and then use it